app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -20,9 +20,16 @@ def proxy_command_options(template_invocation, host)
           super(template_invocation, host).merge(
             'ansible_inventory' => ::ForemanAnsible::InventoryCreator.new(
               [host], template_invocation
-            ).to_hash.to_json
+            ).to_hash.to_json,
+            :remote_execution_command => ansible_command?(template_invocation.template)
           )
         end
+
+        private
+
+        def ansible_command?(template)
+          template.remote_execution_features.where(:label => 'ansible_run_host').empty?
```

Comment:
Shouldn't this line be `.present?`? Right now it's disabling the callback for *all* runs, except for ones related to the 'ansible run host' feature.

Reply:
It is correct to check if there are `any?` features with `ansible_run_host`. When the feature would be present? it would not be a `ansible_command?` and therefore not disable callback. Only those using the feature should have this set to "false" and "enable" the callback, by not setting the environment variable to empty.